artifacts/README.md

```diff
@@ -1,10 +1,16 @@
 # Layer static spec file
-[home](../README.md)
-* The artifact hold the entire layer spec and provided on layer creation as json body.
+[Home](../README.md)
+* Artifact holds layer spec and is provided on layer creation as json body.
 * On current version it support hard-coded only file 'polygonPartSpec.json'.
-* used for featurestypes rest APIs [/workspaces](https://docs.geoserver.org/latest/en/api/#1.0.0/featuretypes.yaml).
+* Used for featurestypes rest APIs [/workspaces](https://docs.geoserver.org/latest/en/api/#1.0.0/featuretypes.yaml).
 * Not supports xml format on current version.
 
+> [!NOTE]
+> On current version it support hard-coded only file 'polygonPartSpec.json'.
+
+> [!NOTE]
+> Not supports xml format on current version.
```

Comment:
Current version doesn't supports xml format

Reply:
done.
is otherwise it putting it on same line, doesn't support me the option of list